app/src/main/java/net/skyscanner/backpack/demo/compose/ThemeStory.kt

```diff
@@ -0,0 +1,170 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.gestures.Orientation
+import androidx.compose.foundation.gestures.scrollable
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.BottomAppBar
+import androidx.compose.material.Button
+import androidx.compose.material.Card
+import androidx.compose.material.Checkbox
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.ExtendedFloatingActionButton
+import androidx.compose.material.FloatingActionButton
+import androidx.compose.material.Icon
+import androidx.compose.material.LinearProgressIndicator
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Slider
+import androidx.compose.material.Snackbar
+import androidx.compose.material.Switch
+import androidx.compose.material.Tab
+import androidx.compose.material.TabRow
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.flowlayout.FlowCrossAxisAlignment
+import com.google.accompanist.flowlayout.FlowMainAxisAlignment
+import com.google.accompanist.flowlayout.FlowRow
+import net.skyscanner.backpack.compose.icons.BpkIcons
+import net.skyscanner.backpack.compose.icons.lg.Flight
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkElevation
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun ThemeStory() {
+  FlowRow(
+    modifier = Modifier
+      .scrollable(state = rememberScrollState(), orientation = Orientation.Horizontal)
+      .padding(all = BpkSpacing.Base),
+    mainAxisSpacing = BpkSpacing.Base,
+    mainAxisAlignment = FlowMainAxisAlignment.Center,
+    crossAxisSpacing = BpkSpacing.Base,
+    crossAxisAlignment = FlowCrossAxisAlignment.Center,
+  ) {
+
+    TopAppBar {
+      Text(
+        modifier = Modifier.padding(horizontal = BpkSpacing.Base),
+        text = "Top app bar",
```

Comment:
We're going to remove this story once each of this components will be implemented in Backpack. Therefore strings here are mostly Kotlin-strings.